contracts/amm.cairo

```diff
@@ -119,13 +114,21 @@ func do_trade{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
 
         // 3) Calculate new volatility, calculate trade volatility
         
-        let (current_pool_balance) = get_pool_available_balance(lptoken_address);
-        assert_nn_le(1, current_pool_balance);
+        let (current_pool_balance) = get_unlocked_capital(
```

Comment:
Hmm, I changed get_pool_available_balance to get_unlocked_capital, should check later

Reply:
Oh no that isn't it, get_pool_available_balance was a function that only called get_unlocked_capital and I deleted it because it's useless.